tests/robustness/multiclient_test/multiclient_test.go

```diff
@@ -121,6 +140,12 @@ func TestManySmallFilesAcrossDirecoryTree(t *testing.T) {
 
 		_, err = eng.ExecAction(ctx, engine.RestoreSnapshotActionKey, snapOut)
 		require.NoError(t, err)
+
+		_, err = eng.ExecAction(ctx, engine.DeleteRandomSubdirectoryActionKey, deleteDirOpts)
+		require.NoError(t, err)
+
+		_, err = eng.ExecAction(ctx, engine.DeleteDirectoryContentsActionKey, deleteDirOpts)
+		require.NoError(t, err)
```

Comment:
Same question as above: what is the effect of performing these operations here?

Reply:
The robustness tests don't test the deleting directories or directory content at the moment. The intent is to provide the coverage for the missing actions.
The current sequence of actions, deletion followed by data generation, reduces the snapshot size growth, thus ensuring the reduction in capacity required for long term testing.